packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -93,7 +93,7 @@ public function __construct( $options = array() ) {
 		$this->mapping['term_id']   = array();
 		$this->requires_remapping   = $empty_types;
 		$this->exists               = $empty_types;
-		$this->logger               = new Logger();
+		$this->logger               = isset( $options['logger'] ) ? $options['logger'] : new WP_Logger();
```

Comment:
I'm actually not sure we need that. All the progress information are now exposed as numbers and the API consumer may implements its own logging using any technique.

Reply:
Make sense, thanks.